profiles/bicycle.lua

```diff
@@ -367,35 +368,73 @@ function handle_bicycle_tags(profile,way,result,data)
   end
 
   -- cycleway
-  local has_cycleway_left = false
-  local has_cycleway_right = false
+  local has_cycleway_forward = false
+  local has_cycleway_backward = false
+
+  -- cycleways on normal roads
+  if result.forward_mode ~= mode.inaccessible and result.backward_mode ~= mode.inaccessible and not implied_oneway then
```

Comment:
We could capture this condition in a variable for documentation. Maybe something like `local is_bidirectional = ....`